source/renderer/app/components/wallet/WalletCreateDialog.js

```diff
@@ -142,6 +142,9 @@ export default class WalletCreateDialog extends Component<Props, State> {
           validators: [
             ({ field, form }) => {
               const repeatPasswordField = form.$('repeatPassword');
+              if (repeatPasswordField.value.length === 0) {
+                repeatPasswordField.validate({ showErrors: false });
+              }
               if (repeatPasswordField.value.length > 0) {
                 repeatPasswordField.validate({ showErrors: true });
               }
```

Comment:
@DeeJayElly Can you isolate `repeatPasswordField ` length and then just add something like 
`repeatPasswordField.validate({ showErrors: isRepeatPasswordFieldSet });` without multiple if clauses?

Reply:
Yep it's done.